module/hooks/action.mjs

```diff
@@ -63,6 +63,15 @@ HOOKS.assessStrength = {
 
 /* -------------------------------------------- */
 
+HOOKS.beastShapeRevert = {
```

Comment:
Comment: `beastShape` might be a good candidate to refactor into a "Maintained" action once we have that mechanic implemented.